helpdesk_mgmt/models/helpdesk_ticket.py

```diff
@@ -243,12 +249,6 @@ def _message_get_suggested_recipients(self):
                     ticket._message_add_suggested_recipient(
                         recipients, partner=ticket.partner_id, reason=_("Customer")
                     )
-                elif ticket.partner_email:
```

Comment:
Why removing this? Is it suggested by other means?

Reply:
This is the code for the partner creation from the chatter, the preceding message explains why this could be problematic to keep it

Reply:
I have just tried on runboat and now the mail is not suggested to be created, so that's incorrect. Please check how it's done in CRM, that the mail is suggested to be added and the partner created. You have to worry about how to transfer all the data from the helpdesk ticket form to the created partner, the same as CRM does.

Reply:
Can you point me to an example of what I should copy ? I've tested on the CRM quote form and it did not do what I expected.

Indeed the partner is suggested to be added, but the partner it creates does not transfer the data from the quote and is not attached to the quote either.

Reply:
Why you talk about quote? I'm talking about opportunity and sending a message from it. The recipient is suggested, and when you create the partner, all data is correctly put on the created partner.

Reply:
Sorry, I was talking about the opportunity, not the quote.

I made a mistake, the partner is attached to the opportunity once the message is sent.

But the data is not put on the created partner. Here is a video of what I did (on runbot.odoo.com 14.0)

https://user-images.githubusercontent.com/8819682/190999449-f9e38806-48f3-4f05-9083-82a5482a5f5e.mp4

The data is no passed from the "phone" field. I did the test with the fields in "extra information" (Contact Name, job position & mobile) and the data is not passed either.

Reply:
https://user-images.githubusercontent.com/8819682/190999856-b4e8b4ce-913d-4dd6-8289-cf28ed3a41ca.mp4

Reply:
Looks like github  dos not like my file, so here is a download link: https://upload.disroot.org/r/1Etx1bSy#kNF14n1+Xnr0DxwZAY5AlYaqIszYIxQi1Uax8S2a7sY=

Reply:
I can indeed open an issue to notify Odoo of this problem, I agree that this should be the expected behavior.

The thing is, looking at how this is done for now, this would be more of the implementation of a functionality than a bugfix. 

For now the chatter calls odoo to get the suggested recipients (https://github.com/odoo/odoo/blob/14.0/addons/mail/controllers/main.py#L314)
When Odoo answers, it passes a partner id and an email, there is no place for a list of values (https://github.com/odoo/odoo/blob/14.0/addons/mail/models/mail_thread.py#L1515)
Also, in the JS side, there is nothing other than the email that is expected, even the name is created from the email field (https://github.com/odoo/odoo/blob/14.0/addons/mail/static/src/models/thread/thread.js#L605 )

That would mean a lot of refactoring of `mail` module to make it work for the helpdesk.

It should indeed be easy to add a line in the partner creation form (https://github.com/odoo/odoo/blob/14.0/addons/mail/static/src/components/composer_suggested_recipient/composer_suggested_recipient.xml#L24 ) but as there is no way to communicate the data to put in suggestedRecipientInfo it is not feasible for now.

Reply:
I've added a workaround inspired by [Opportunity to quotation](https://github.com/odoo/odoo/blob/14.0/addons/sale_crm/wizard/crm_opportunity_to_quotation.py) to send the email while making sure the partner exists